libnavigation-base/src/main/java/com/mapbox/navigation/base/options/RoutingTilesOptions.kt

```diff
@@ -22,7 +22,9 @@ import java.net.URI
  * @param filePath used for persistent configuration and history files storing.
  * @param tileStore tile store instance. It manages downloads and storage for requests to
  * tile-related API endpoints. For offline/predictive-caching use cases this instance should be
- * the same that is passed to map resource options.
+ * the same that is passed to map resource options. When creating the [TileStore] make sure to call
+ * [TileStore.setOption] with [TileStoreOptions.MAPBOX_ACCESS_TOKEN] and your token.
+ * By default (if `null` is provided here), the [TileStore] will be created with a [filePath] and [NavigationOptions.accessToken].
```

Comment:
should we mention domain?
`TileDataDomain.NAVIGATION`

Reply:
I don't think we have to. It should be very a rare occasion where a developer needs to specify a domain, normally, they can use the same token for all of their usages.